changelog: 2025-04-02-sebankid-accessibility (#174)
criipto/docs: src/pages/changelog/2025-04-02-sebankid-accessibility.mdx

---
date: 2025-04-02
title: Stronger accessibility requirements for Swedish BankID
---

From July 2025, stronger requirements regarding accessibility must be fulfilled to meet legal demands.
Amongst other things, these will affect the two-device flow for BankID where the animated QR code is used.

> [Learn more about new requirements](https://developers.bankid.com/getting-started/qr-code#accessibility)

Reviewer: ```suggestion
[Learn more about the new requirements](https://developers.bankid.com/getting-started/qr-code#accessibility)
```

> In order to comply with the new QR code UI requirements Criipto must make adjustments to its UI (HTML & CSS),

Reviewer: ```suggestion
In order to comply with the new QR code UI requirements Criipto must make adjustments to our UI (HTML & CSS),
```

so that users may click the QR code to view it in fullscreen.

> This can cause breaking changes to customers who have custom CSS, so customers must explicitely opt-in to the UI update.

Reviewer: ```suggestion
This can cause breaking changes to customers who have custom CSS, so customers must explicitly opt-in to the UI update.
```

> This can cause breaking changes to customers who have custom CSS, so customers must explicitly opt-in to the UI update.
> 

Reviewer: This might make it sound like customers have an option not to do it. Maybe add something like:
```suggestion
This can cause breaking changes to customers who have custom CSS, so customers must explicitly enable the UI update. To comply with the new requirements, you must enable the UI update before July 2025.

```
